benchmarks/automated/README.md

```diff
@@ -50,12 +50,12 @@ aws iam attach-role-policy --policy-arn arn:aws:iam::aws:policy/AmazonEC2FullAcc
 aws iam attach-role-policy --policy-arn arn:aws:iam::aws:policy/AmazonS3FullAccess --role-name EC2Admin
 aws iam attach-role-policy --policy-arn arn:aws:iam::aws:policy/AmazonEC2ContainerRegistryFullAccess --role-name EC2Admin
 ```
--- S3_BUCKET_BENCHMARK_ARTIFACTS :all temporary benchmarking artifacts including server logs will be stored in this bucket: <br>
+-- s3_bucket_benchmark_artifacts :all temporary benchmarking artifacts including server logs will be stored in this bucket: <br>
 Use the following command to create a new S3 bucket if you don't have one you can use.
 ```
 aws s3api create-bucket --bucket <torchserve-benchmark> --region us-west-2
 ```
--- DEFAULT_DOCKER_DEV_ECR_REPO :docker image used for benchmarking will be pushed to this repo <br>
+-- default_docker_dev_ecr_repo :docker image used for benchmarking will be pushed to this repo <br>
 Use the following command to create a new ECR repo if you don't have one you can use.
 ```
 aws ecr create-repository --bucket torchserve-benchmark --region us-west-2
```

Comment:
@nksol "aws ecr create-repository --bucket torchserve-benchmark --region us-west-2" is not correct

Reply:
Thanks, corrected.